packages/shared/wallet/src/wallet-selector/wallet-selector.tsx

```diff
@@ -55,12 +56,19 @@ export function WalletSelector() {
     }
   };
 
+  useEffect(() => {
```

Comment:
I wonder if this is the right place to put these intervals

Reply:
Move this stuff into the use-connect hook

Reply:
I think this is fine tbh